index.js

```diff
@@ -75,5 +75,33 @@ function handleLine(line) {
         }
     } else if (line.indexOf("/reload-partitions") === 0) {
         sv.reloadPartitionMap();
+    } else if (line.indexOf("/globalban") === 0) {
+        var args = line.split(" "); args.shift();
+        // simple validation for IPv4
+        // TODO: IPv6
+        if(args.length === 2 && args[0].match(/^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$/)){
+            require("./lib/database").globalBanIP(args[0], args[1], function (err, res) {
+                if (!err) {
+                    Logger.eventlog.log("[acp] " + "SYSTEM" + " global banned " + args[0]);
+                }
+            })
+        }
+    } else if (line.indexOf("/clearban") === 0) {
```

Comment:
/unglobalban or something like that would be more consistent naming.

Replace the IP address regex with `net.isIPv4()`: https://nodejs.org/api/net.html#net_net_isipv4_input